lib/mj-batik.js

```diff
@@ -0,0 +1,28 @@
+var fs = require('fs');
+var os = require('os');
+var path = require('path');
+var execFile = require('child_process').execFile;
+
+exports.svg2png = function(svg, result, scale, width, height, callback, check) {
+  var tmpfile = os.tmpdir() + "/mj-svg" +  process.pid;  // file name prefix to use for temp files
+  var BatikRasterizerPath = path.resolve(__dirname,'..','batik/batik-rasterizer.jar');
+  var batikCommands = ['-jar', BatikRasterizerPath, '-dpi', scale*72, tmpfile + '.svg'];
```

Comment:
You've lost the use of the `dpi` setting that is part of the current API by changing it to `scale*72` here, so that is a breaking change.  (Also, the default dpi passed by the command-line tools is based on the ex size, with the default ex of 6 producing a dpi of 96, so this will produce different sized images from the current version.  I worked a bit to get the `dpi` values to do the right thing for the given ex size.)